justfile

```
@@ -0,0 +1,12 @@
+_default:
+  @just --list
+
+build:
+  #!/usr/bin/env bash
+  whiskers fcitx5.tera
+  whiskers --list-flavors -o plain | while read -r flavor; do
+    whiskers --list-accents -o plain | while read -r accent; do
```

```suggestion
  for flavor in $(whiskers --list-flavors -o plain ); do
    for accent in $(whiskers --list-accents -o plain ); do
```
Oops, I think this simpler for loop should work!

---

Right, then the simpler loop it shall be!